src/main/java/org/jenkinsci/backend/ircbot/IrcBotImpl.java

```diff
@@ -276,6 +279,21 @@ private void handleDirectCommand(String channel, String sender, String login, St
         }
     }
 
+    private void sendFallbackMessage(String channel, String payload) {
+        if(StringUtils.containsIgnoreCase(payload, "thank")) {
+            sendMessage(channel, "You're welcome");
+        }
+        if(StringUtils.startsWithIgnoreCase(payload, "hello")) {
+            sendMessage(channel, "Hello you");
```

Comment:
You could return name here. 
But two bots may start infinitely talking to each other in such case